components/dropdown/dropdown-content-styles.js

```diff
@@ -0,0 +1,153 @@
+				top: calc(100% + var(--d2l-dropdown-verticaloffset, 20px));
+				width: 100%;
+				z-index: 1000; /* position on top of floating buttons */
+			}
+
+			:host([opened]) {
+				display: inline-block;
+			}
+
+			:host([opened]) {
+				-webkit-animation: d2l-dropdown-animation 300ms ease;
+				animation: d2l-dropdown-animation 300ms ease;
+			}
+
+			:host([opened-above]) {
+				bottom: calc(100% + var(--d2l-dropdown-verticaloffset, 20px));
+				top: auto;
+				-webkit-animation: d2l-dropdown-above-animation 300ms ease;
+				animation: d2l-dropdown-above-animation 300ms ease;
+			}
+
+			:host .d2l-dropdown-content-pointer {
+				position: absolute;
+				display: inline-block;
+				clip: rect(-5px, 21px, 8px, -7px);
+				top: -7px;
+				left: calc(50% - 7px);
+				z-index: 1;
+			}
+
+			:host .d2l-dropdown-content-pointer > div {
+				background-color: #ffffff;
+				border: 1px solid var(--d2l-color-mica);
+				border-radius: 0.1rem;
+				box-shadow: -4px -4px 12px -5px rgba(86, 90, 92, .2);
+				height: 16px;
+				width: 16px;
+				transform: rotate(45deg);
+				-webkit-transform: rotate(45deg);
+			}
+
+			:host([opened-above]) .d2l-dropdown-content-pointer {
+				top: auto;
+				clip: rect(9px, 21px, 22px, -3px);
+				bottom: -8px;
+			}
+
+			:host([opened-above]) .d2l-dropdown-content-pointer > div {
+				box-shadow: 4px 4px 12px -5px rgba(86, 90, 92, .2);
+			}
+
+			:host([no-pointer]) .d2l-dropdown-content-pointer {
+				display: none;
+			}
+
+			:host .d2l-dropdown-content-position {
+				border-radius: 0.3rem;
+				display: inline-block;
+				position: absolute;
+			}
+
+			:host .d2l-dropdown-content-width {
+				background-color: #ffffff;
+				border: 1px solid var(--d2l-color-mica);
+				border-radius: 0.3rem;
+				box-shadow: 0 2px 12px 0 rgba(0, 0, 0, 0.15);
+				box-sizing: border-box;
+				min-width: 70px;
+				max-width: 370px;
+				position: absolute;
+				width: 100vw;
+			}
+
+			:host([opened-above]) .d2l-dropdown-content-width {
+				bottom: 100%;
+			}
+
+			:host .d2l-dropdown-content-container {
+				box-sizing: border-box;
+				display: inline-block;
+				max-width: 100%;
+				outline: none;
+				padding: 1rem;
+				vertical-align: top; /* prevents baseline bloat - fix for github issue #173 */
+			}
+
+			:host([no-padding]) .d2l-dropdown-content-container {
+				padding: 0;
+			}
+
+			:host .d2l-dropdown-content-top,
+			:host .d2l-dropdown-content-bottom {
+				height: 5px;
+				position: relative;
+				z-index: 2;
+			}
+
+			:host .d2l-dropdown-content-top {
+				border-top-left-radius: 0.3rem;
+				border-top-right-radius: 0.3rem;
+			}
+
+			:host .d2l-dropdown-content-bottom {
+				border-bottom-left-radius: 0.3rem;
+				border-bottom-right-radius: 0.3rem;
+			}
+
+			:host .d2l-dropdown-content-top-scroll {
+				box-shadow: 0 3px 3px 0 rgba(0, 0, 0, 0.05);
+			}
+
+			:host .d2l-dropdown-content-bottom-scroll {
+				box-shadow: 0 -3px 3px 0 rgba(0, 0, 0, 0.05);
+			}
+
+			:host-context([dir="rtl"]) {

```

Comment:
These `:host-context` and `:host[:dir(rtl)]` selectors should be replaced with the following.  The `RtlMixin` should be extended in the content classes in order to sprout this attribute on the host.
```
:host([dir="rtl"]) {
    ...
}
```
The `:host-context` is not supported going forward, and the `:dir` is a Polymer 3 thing that required a special import.